instance/models/mixins/openedx_config.py

```diff
@@ -147,19 +147,12 @@ def _get_configuration_variables(self):
             # Repositories URLs
             "edx_ansible_source_repo": self.configuration_source_repo_url,
             "edx_platform_repo": self.edx_platform_repository_url,
-            # Temporary: We want a configurable `XQUEUE_SESSION_ENGINE` ASAP but it's not in upstream Ginkgo.
-            "xqueue_source_repo": "https://github.com/open-craft/xqueue",
-            # Temporary: We want a configurable `CERTS_QUEUE_POLL_FREQUENCY` ASAP but it's not in upstream Ginkgo.
-            "CERTS_REPO": "https://github.com/open-craft/edx-certificates",
 
             # Pin down dependencies to specific (known to be compatible) commits.
             "edx_platform_version": self.edx_platform_commit,
             "configuration_version": self.configuration_version,
             "forum_version": self.openedx_release,
-            # Temporary: We want a configurable `XQUEUE_SESSION_ENGINE` ASAP but it's not in upstream Ginkgo.
-            "xqueue_version": 'opencraft-release/ginkgo.1',
-            # Temporary: We want a configurable `CERTS_QUEUE_POLL_FREQUENCY` ASAP but it's not in upstream Ginkgo.
-            "certs_version": "opencraft-release/ginkgo.1",
+            "xqueue_version": self.openedx_release,
```

Comment:
Do we want to also pin the certs_version like this?

Reply:
If it makes sense to, I will add it. I just literally reverted the changes in the PRs which introduced these changes.

Reply:
It probably makes sense I guess, since the default would be `master` right?

Reply:
@UmanShahzad, You are right about the default being `master'. I have updated the PR to pin the `certs_version` to the Open edX release being installed.